cfgov/v1/migrations/0006_update_tables.py

```diff
@@ -36,8 +36,11 @@ def update_full_width_text_tables(wagtail_page):
                 if sub_item['type'] == 'table':
                     sub_item['type'] = 'table_block'
                     sub_item['value'] = create_tableblock_data(old_table=sub_item['value'])
-                    print "Creating a new TableBlock on page %s" %(wagtail_page.title)
+                    print "Creating a new TableBlock on page %s of page type %s" %(wagtail_page.title, wagtail_page.__class__.__name__)
 
+def convert_hyperlink_obj_to_text(value):
+    """ Taken from cfgov/jinja2/v1/_includes/atoms/hyperlink.html """
+    return """<a class='jump-link' href='%s'>%s</a>""" %(value['url'], value['text'])
 
 def create_tableblock_data(old_table):
```

Comment:
Comprehensions are considered more readable and more pythonic. They have lower overhead and are optmized...etc.

```
my_list = []
for item in list2:
    if cond(item):
       my_list.append(operand(item))
```
becomes

```
my_list = [operand(item) for item in list2 if cond(item)] # this reads like english
```